packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useMemo } from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Table } from '@deriv-com/ui';
+import { formattedLoginHistoryData } from '../../pages/LoginHistory/formattedLoginHistoryData';
+
+type TLoginHistoryData = {
```

Comment:
```suggestion
type TLoginHistory = {
```


```suggestion
type TLoginHistoryProps = {
```

please update the name either with one of these are this is a component props type